pylast/__init__.py

```diff
@@ -42,16 +42,34 @@
 def _deprecation_warning(message):
     warnings.warn(message, DeprecationWarning)
 
+
+def _can_use_ssl_securely():
+    # Python 3.3 doesn't support create_default_context() but can be made to
+    # work sanely.
+    # <2.7.9 and <3.2 never did any SSL verification so don't do SSL there.
+    # >3.4 and >2.7.9 has sane defaults so use SSL there.
+    v = sys.version_info
+    return v > (3, 3) or (v < (3, 0) and v > (2, 7, 9))
```

Comment:
Looks like clever code but I'll allow it. Interval comparison looks neat (but clever)!